internal/gengapic/options.go

```diff
@@ -72,6 +74,9 @@ func parseOptions(parameter *string) (*options, error) {
 		// check for the boolean flag, sample-only, that disables client generation
 		if s == "sample-only" {
 			return &options{sampleOnly: true}, nil
+		} else if s == "metadata" {
```

Comment:
We can remove the `else` and outdent the `if` statement, right? Seems more idiomatic and clearer

Reply:
Sure thing. Done!